### ISSUE TITLE
New static page for "In the media"

### ISSUE THREAD

**Issue by frrrances** [MITLibraries/MITLibraries-news#37]
Brigham needs a page to keep a list of references of where the library has been in the news. For now, we would like to give her a single static page where she can add/edit the content. 

It should be linked in the "About us" in the main nav, listed as "In the media" under "Exhibits & Galleries"

**Comment by matt-bernhardt** [comment]
This appears to be a problem with the body area not appearing on libraries-test for admins. We're investigating.

Dev and Production seem to allow the editor to see the body field, but not Test.

**Comment by PBruk** [comment]
This took me a while to figure out, as well... At first I thought it was something in the functions.php file, but after going thru the entire file, I realized it had to be something else. The culprit? The ACF Field Group of Post Author. If you click on that field group and scroll to the bottom, you will see a section called "Hide on screen," which has text editor, among many other options selected... I have now unselected it in on Test, as well.
